__tests__/app.test.js

```diff
@@ -467,4 +466,22 @@ describe("PATCH/api/articles/:article_id", () => {
   });
 });
 
-      
+describe("(GET/api/users)", () => {
+  test("200: should return array of all the user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users).toBeInstanceOf(Array);
```

Comment:
Please check the length of the array here!

Reply:
Thanks, I just amended the test. Going to merge it now!